201-cosmosdb-advanced-threat-protection-create-account/azuredeploy.json

```diff
@@ -0,0 +1,70 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "name": {
+      "type": "string",
+      "defaultValue": "[uniqueString('cosmosdb', resourceGroup().id)]",
+      "metadata": {
+        "description": "The name of your CosmosDB account"
+      }
+    },
+    "location": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().location]",
+      "metadata": {
+        "description": "Location for your CosmosDB account"
+      }
+    },
+    "tier": {
+      "type": "string",
+      "defaultValue": "Standard",
+      "metadata": {
+        "description": "CosmosDB account tier"
+      }
+    },
+    "advancedThreatProtectionEnabled": {
+      "type": "bool",
+      "defaultValue": true,
+      "metadata": {
+        "description": "Enable or disable Advanced Threat Protection."
+      }
+    }
+  },
+  "resources": [
+    {
+      "name": "[parameters('name')]",
+      "type": "Microsoft.DocumentDB/databaseAccounts",
+      "apiVersion": "2016-03-31",
```

Comment:
are you sure about the API version?

Reply:
same as all other cosmos db templates

Reply:
@bmoore-msft this is the latest API version, but the CI pipeline fails with
`apiVersions-Should-Be-Recent.test.ps1 : Api versions should be under 2 years old (730 days) - (Microsoft.DocumentDB/databaseAccounts is 1202 days old)` 
can we use this version although it's old? (as it's the most recent)

Reply:
yes